server/entity/splashable.go

```diff
@@ -66,10 +80,25 @@ func potionSplash(durMul float64, pot potion.Potion, linger bool) func(e *Ent, r
 					if h := blockPos.Side(f); w.Block(h) == fire() {
 						w.SetBlock(h, nil, nil)
 					}
+
+					if b, ok := w.Block(blockPos.Side(f)).(SplashableBlock); ok {
+						b.Splash(w, blockPos.Side(f), pot)
+					}
+				}
+
+				resultPos := result.BlockPosition()
+				if b, ok := w.Block(resultPos).(SplashableBlock); ok {
+					b.Splash(w, resultPos, pot)
 				}
 			case trace.EntityResult:
 				// TODO: Damage endermen, blazes, striders and snow golems when implemented and rehydrate axolotls.
 			}
+
+			for _, otherE := range w.EntitiesWithin(box.Grow(3), ignores) {
```

Comment:
Where is this bounding box size coming from? Shouldn't this match the above used for splash potions with effects?